docs/bosh/README.md

```diff
@@ -398,13 +451,41 @@ Now you have the infrastructure ready to deploy a BOSH director.
 terminal multiplexer such as `tmux` or `screen`.
 
    ```
-   bosh-init deploy manifest.yml
+   bosh2 create-env manifest.yml
    ```
 
-1. Target your BOSH environment:
+1. Use `vim` or `nano` to create a certificate named `ca_cert.pem`:
+
+    ```
```

Comment:
Your comment got lost when you squashed. I guess it's best to not squash on github and leave the history of the review.
```
The certificate is now pasted in like the director manifest. Using bosh2 int would require copying the generated certs.yml that correspond to the hardcoded certificates in the director manifest along with the bosh2 invocation.

I've included a note which provides the ability to delve deeper into certificate handling.
```

You can use bosh2 int to extract a value from an existing manifest. 

eg:
```bash
echo "foo: bar" > foo.yml
bosh2 int foo.yml --path /foo
```